app/src/main/java/com/sma2/sma2/ExerciseFragments/ExSustainedVowel.java

```diff
@@ -0,0 +1,187 @@
+package com.sma2.sma2.ExerciseFragments;
+
+import android.app.Activity;
+import android.content.Context;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Message;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ProgressBar;
+
+import com.sma2.sma2.R;
+import com.sma2.sma2.SignalRecording.SpeechRecorder;
+
+import java.io.File;
+
+
+/**
+ * A simple {@link Fragment} subclass.
```

Comment:
Maybe we should remove all the generiv comments. I think they should add confusion